Make trailing spaces pop out more

elisee opened the pull request:
`@background-color-selected` doesn't stand out in most themes, and it's easily confused with a selection.

This pull request uses `@background-color-error` instead which is red in One Dark. Alternatively, `@background-color-warning` could be used if you consider `@background-color-error` too much.

As for me, I like my trailing spaces to be easy targets so I can take them out swiftly! :dart:  thus ensuring they never make it to source control.

PS: Oh, and after making the change, I noticed there were pesky trailing spaces in `trailing-spaces.less` :wink:

kankaristo commented:
> PS: Oh, and after making the change, I noticed there were pesky trailing spaces in trailing-spaces.less

Depends on what you consider as "trailing". ;)

Personally, I don't consider indentation (leading whitespace) to be trailing whitespace, which is why I added the "Enable For Indentation" setting.

I like to keep indentation consistent for empty lines as well, so the cursor doesn't jump around when moving around with the keyboard and so that it doesn't break the lines on the indent guide.

> @background-color-selected doesn't stand out in most themes, and it's easily confused with a selection.

I'm okay with changing the color, but it would be nice to have a custom variable for this, which the user could change in their `styles.less` and would only *default* to `@background-color-error` or `@background-color-warning` (see https://github.com/wpillar/atom-trailing-spaces/issues/14).

This way it would work with all themes, and the theme could actually define it's own color (unless the user overrides it).

I think the error color is a little too eye-catching, especially since some files will have indentation as "trailing" whitespace, and that may be intentional (even if your settings say otherwise). So maybe `@background-color-warning` would be better. :+1:

kankaristo commented:
Although, I'm not sure if a custom variable is even possible (without importing all of `styles.less`).

elisee commented:
> Depends on what you consider as "trailing". ;)
> 
> Personally, I don't consider indentation (leading whitespace) to be trailing whitespace, which is why I added the "Enable For Indentation" setting.

Fair enough, restored the indentation and switched to `@background-color-warning`. A setting /  custom variable would be great but I have no idea how to do that.

kankaristo commented:
Yeah, I guess the custom color could be left for another day. :)

The new color looks good. It's 1 AM here, so I'll publish this tomorrow. :+1:

kankaristo commented:
@elisee, I decided to do this right away instead, but `apm publish patch` was doing something horrible (might be a bug in `apm`).

Could you check that the new version (0.3.2) is published and you can update it in Atom?

elisee commented:
Looks like it updated fine on my end. Thanks!
